src/tss2-mu/tss2-mu.vcxproj

```diff
@@ -43,12 +43,12 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'" Label="Configuration">
     <ConfigurationType>DynamicLibrary</ConfigurationType>
     <UseDebugLibraries>true</UseDebugLibraries>
-    <PlatformToolset>v141_clang_c2</PlatformToolset>
```

Comment:
Looks like the first patch changes some of these and patch two changes the test in the same file. Is there a reason why these need to be in separate patches. If not please squash them together.

Reply:
The commits are now squashed